src/test/e2e/api/admin/export-import.e2e.test.ts

```diff
@@ -71,11 +98,34 @@ test('exports features', async () => {
         .set('Content-Type', 'application/json')
         .expect(200);
 
+    const { name, ...resultStrategy } = strategy;
     expect(body).toMatchObject({
         features: [
             {
                 name: 'first_feature',
             },
         ],
+        featureStrategies: [resultStrategy],
+    });
+});
+
+test('returns all features, when no feature was defined', async () => {
```

Comment:
suggested rename => returns all features by default

I'd also consider: features not specified at all or set to undefined